pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
```

Comment:
I don't think this pyproject file works. it didn't locally on osx with py38 and with "pip install -e ".[all]"", granted I removed the lines I commented on below. The error I'm getting is relatd to what looks like pandas, where pip dowjnloads like all possible versions, trying to find one that works, and then fails

```bash
...snip... (downloads all other version of pandas up to like 2.1.0 or whatever is current)

  Preparing metadata (setup.py) ... done
  Downloading pandas-0.24.2.tar.gz (11.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 11.8/11.8 MB 71.1 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [19 lines of output]
      /private/var/folders/30/31lw1t9s7d7_1xcc8b5x2rgw0000gp/T/pip-install-yk42uovx/pandas_c6380d1be0594b759ab90a795ca6f885/setup.py:12: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
        import pkg_resources
      /private/var/folders/30/31lw1t9s7d7_1xcc8b5x2rgw0000gp/T/pip-install-yk42uovx/pandas_c6380d1be0594b759ab90a795ca6f885/setup.py:435: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
        current_system = LooseVersion(platform.mac_ver()[0])
      /private/var/folders/30/31lw1t9s7d7_1xcc8b5x2rgw0000gp/T/pip-install-yk42uovx/pandas_c6380d1be0594b759ab90a795ca6f885/setup.py:436: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
        python_target = LooseVersion(
      /Users/dtmcdonald/micromamba/envs/mg-scripts/lib/python3.8/site-packages/setuptools/__init__.py:80: _DeprecatedInstaller: setuptools.installer and fetch_build_eggs are deprecated.
      !!
      
              ********************************************************************************
              Requirements should be satisfied by a PEP 517 installer.
              If you are using pip, you can try `pip install --use-pep517`.
              ********************************************************************************
      
      !!
        dist.fetch_build_eggs(dist.setup_requires)
      error in pandas setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Expected end or semicolon (after version specifier)
          pytz >= 2011k
               ~~~~~~~^
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

Reply:
reverting back to the existing setup.py works other than changing the readme file extension

Reply:
That's odd. I actually wrote pyproject.toml to specifically address the issue of installing mg-scripts and its dependency metapool into brand new environments and tested it before the commit. It's better to use pyproject.toml as mg-scripts is a dependency of qp-klp which itself uses pyproject.toml and can encounter issues if mg-scripts uses setup.py.

Reply:
File removed.